package.json

```diff
@@ -17,15 +36,16 @@
   "author": "Developer Circles Community",
   "license": "ISC",
   "bugs": {
-    "url": ""
+    "url": "https://github.com/DevCEldoret/devceldoret-backend/issues"
   },
   "homepage": "",
   "dependencies": {
     "body-parser": "^1.19.0",
     "cors": "^2.8.5",
-    "dotenv-safe": "^8.2.0",
+    "dotenv": "^8.2.0",
```

Comment:
The reason we were using `dotenv-safe` instead of `dotenv` is to be able to have a `.env.example` file that will guide contributors on what keys are required in the `.env` file

Reply:
sure